### PR TITLE
78336 internal server error code functoinal test for mock SAP API

### PR DESCRIPTION
**Pull request by prashantmgupta**

### CODE REVIEW

File: tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/WebhookScenarios.cs
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/WebhookScenarios.cs:1]: Please change file name

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/TestConfig.cs [hunk 1]
```diff
@@ -4,9 +4,11 @@ namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
     public class TestConfig
     {
         public string PayloadFolder { get; set; }
-        public string WebhookPayloadFileName { get; set; }
+        public string WebhookPayloadFileName { get; set; } 
+        public string SapApiPayloadFileName { get; set; }
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/TestConfig.cs:8]: Same as above

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/TestConfig.cs [hunk 2]
```diff
@@ -4,9 +4,11 @@ namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
...
         public Erpfacadeconfiguration ErpFacadeConfiguration { get; set; }
-        public Azureadconfiguration AzureadConfiguration { get; set; }
+        public Azureadconfiguration AzureadConfiguration { get; set; } 
+        public SapConfiguration SapConfiguration { get; set; }
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/TestConfig.cs:11]: Same as above

File: tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
   "PayloadFolder": "ERPFacadePayloadTestData",
   "WebhookPayloadFileName": "WebhookPayload.json",
+  "SapApiPayloadFileName": "SapApiConfigure.xml",
 
   "ErpFacadeConfiguration": {
-    "BaseUrl": ""
+    "BaseUrl": "https://localhost:65117"
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json:7]: Remove URL

**Review by Sadha1501493 — CHANGES_REQUESTED** [review]

**Review by RaviK2233 — APPROVED** [review]
Reviewed FT and looks good.

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**